tests/testthat/test_environment.R

```diff
@@ -20,6 +20,12 @@ test_that("create environment and check parameters", {
   expect_equal(env$docker$enabled, TRUE)
   expect_equal(env$docker$base_dockerfile, NULL)
   expect_equal(env$docker$base_image, custom_docker_image_name)
+
+  # use extra packages
+  env <- r_environment(env_name, cran_packages = c("ggplot2", "dplyr"))
+  expect_equal(length(env$r$cran_packages), 2)
+  expect_equal(env$r$cran_packages[[1]]$name, "ggplot2")
+  expect_equal(env$r$cran_packages[[2]]$name, "dplyr")
 })
 
 test_that("create, register, and get environment", {
```

Comment:
can you also add a test for github_packages and custom_url_packages?

Reply:
and bioconductor package?

Reply:
added